### ISSUE TITLE
Getting this in the Windows App Store

### ISSUE THREAD

**Issue by devinrhode2** [taviso/hotcorner#22]
Hi I was curious if I could put this on the windows app store or help an owner/maintainer do so

**Comment by taviso** [comment]
Sure, I don't see why not. If you want to submit it, please just make sure there is a link to the github repo.

Are there any changes required?

**Comment by devinrhode2** [comment]
Not sure, I haven't gotten into it at all yet, but I will soon! I'll try
and keep this thread updated

**Comment by rocketinventor** [comment]
This sounds like a great idea!

@devinrhode2 Any update on this?

**Comment by devinrhode2** [comment]
No, nothing ever happened here, I moved off windows pretty quickly
